src/Orchard.Web/Modules/Orchard.OutputCache/Filters/OutputCacheFilter.cs

```diff
@@ -314,6 +314,12 @@ public class OutputCacheFilter : FilterProvider, IActionFilter, IResultFilter, I
                 return false;
             }
 
+            // Don't cache ignored routes
+            if (_cacheRouteConfig != null && _cacheRouteConfig.Duration == 0) {
+                Logger.Debug("Request for item '{0}' ignored because route is configured to not be cached.", itemDescriptor);
```

Comment:
Would be nice to include the route in the log message here, like so:

    Request for item '{0}' ignored because the route '{1}' is configured to not be cached.